boards/common/stm32/include/l4/cfg_clock_default.h

```diff
@@ -0,0 +1,187 @@
+#endif
+#endif /* CONFIG_USE_CLOCK_PLL */
+
+#ifndef CONFIG_USE_CLOCK_MSI
+#define CONFIG_USE_CLOCK_MSI            0
+#endif /* CONFIG_USE_CLOCK_MSI */
+
+#ifndef CONFIG_USE_CLOCK_HSE
+#define CONFIG_USE_CLOCK_HSE            0
+#endif /* CONFIG_USE_CLOCK_HSE */
+
+#ifndef CONFIG_USE_CLOCK_HSI
+#define CONFIG_USE_CLOCK_HSI            0
+#endif /* CONFIG_USE_CLOCK_HSI */
+
+#if IS_ACTIVE(CONFIG_USE_CLOCK_PLL) && \
+    (IS_ACTIVE(CONFIG_USE_CLOCK_MSI) || IS_ACTIVE(CONFIG_USE_CLOCK_HSE) || \
+     IS_ACTIVE(CONFIG_USE_CLOCK_HSI))
+#error "Cannot use PLL as clock source with other clock configurations"
+#endif
+
+#if IS_ACTIVE(CONFIG_USE_CLOCK_MSI) && \
+    (IS_ACTIVE(CONFIG_USE_CLOCK_PLL) || IS_ACTIVE(CONFIG_USE_CLOCK_HSI) || \
+     IS_ACTIVE(CONFIG_USE_CLOCK_HSE))
+#error "Cannot use MSI as clock source with other clock configurations"
+#endif
+
+#if IS_ACTIVE(CONFIG_USE_CLOCK_HSE) && \
+    (IS_ACTIVE(CONFIG_USE_CLOCK_PLL) || IS_ACTIVE(CONFIG_USE_CLOCK_MSI) || \
+     IS_ACTIVE(CONFIG_USE_CLOCK_HSI))
+#error "Cannot use HSE as clock source with other clock configurations"
+#endif
+
+#if IS_ACTIVE(CONFIG_USE_CLOCK_HSI) && \
+    (IS_ACTIVE(CONFIG_USE_CLOCK_PLL) || IS_ACTIVE(CONFIG_USE_CLOCK_MSI) || \
+     IS_ACTIVE(CONFIG_USE_CLOCK_HSE))
+#error "Cannot use HSI as clock source with other clock configurations"
+#endif
+
+#ifndef CONFIG_BOARD_HAS_HSE
+#define CONFIG_BOARD_HAS_HSE            0
+#endif
+#ifndef CLOCK_HSE
+#define CLOCK_HSE                       MHZ(8)
+#endif
+#if IS_ACTIVE(CONFIG_BOARD_HAS_HSE) && (CLOCK_HSE < MHZ(4) || CLOCK_HSE > MHZ(48))
+#error "HSE clock frequency must be between 4MHz and 48MHz"
+#endif
+
+#ifndef CONFIG_BOARD_HAS_LSE
+#define CONFIG_BOARD_HAS_LSE            0
+#endif
+#if IS_ACTIVE(CONFIG_BOARD_HAS_LSE)
+#define CLOCK_LSE                       (1)
+#else
+#define CLOCK_LSE                       (0)
+#endif
+
+#define CLOCK_HSI                       MHZ(16)
+
+#ifndef CONFIG_CLOCK_MSI
+#define CONFIG_CLOCK_MSI                MHZ(48)
+#endif
+
+/* The following parameters configure a 80MHz system clock with PLL as input clock */
+#ifndef CONFIG_CLOCK_PLL_SRC_MSI
+#if IS_ACTIVE(CONFIG_CLOCK_PLL_SRC_HSE) || IS_ACTIVE(CONFIG_CLOCK_PLL_SRC_HSI) || \
+    CONFIG_BOARD_HAS_HSE
+#define CONFIG_CLOCK_PLL_SRC_MSI        0
+#else
+#define CONFIG_CLOCK_PLL_SRC_MSI        1       /* Use MSI an input clock by default */
+#endif
+#endif /* CONFIG_CLOCK_PLL_SRC_MSI */
+#ifndef CONFIG_CLOCK_PLL_SRC_HSE
+#if IS_ACTIVE(CONFIG_BOARD_HAS_HSE)
+#define CONFIG_CLOCK_PLL_SRC_HSE        1
+#else
+#define CONFIG_CLOCK_PLL_SRC_HSE        0
+#endif
+#endif
+#ifndef CONFIG_CLOCK_PLL_SRC_HSI
+#define CONFIG_CLOCK_PLL_SRC_HSI        0
+#endif
+#ifndef CONFIG_CLOCK_PLL_M
+#define CONFIG_CLOCK_PLL_M              (6)
+#endif
+#ifndef CONFIG_CLOCK_PLL_N
+#define CONFIG_CLOCK_PLL_N              (20)
+#endif
+#ifndef CONFIG_CLOCK_PLL_R
+#define CONFIG_CLOCK_PLL_R              (2)
+#endif
```

Comment:
Would be nice to document how these values translate into `CORE_CLOCK` somewhere.

Reply:
And what the resulting default is.

Reply:
The answer is available a few lines below: https://github.com/RIOT-OS/RIOT/pull/14866/files#diff-be460c66a6368d374926ee1708468655R153